<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Denoiser</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(92, 53, 102);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="input">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>381</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Input</string>
    </property>
    <widget class="QRadioButton" name="radio_in_mic">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Microphone</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_in_file">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>File</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="text_in_path">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>.wav file path</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="output">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>381</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QRadioButton" name="radio_out_file">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>file</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_out_net">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Network</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="text_out_path">
     <property name="geometry">
      <rect>
       <x>63</x>
       <y>30</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Path to output file</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="text_out_ip">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Client IP address</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="text_out_port">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>70</y>
       <width>104</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>port</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_start">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_stop">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="test.qrc"/>
 </resources>
 <connections/>
</ui>
